addons/l10n_latam_check/models/account_payment_method.py

```diff
@@ -0,0 +1,15 @@
+from odoo import models, api
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class AccountPaymentMethod(models.Model):
+    _inherit = 'account.payment.method'
+
+    @api.model
+    def _get_payment_method_information(self):
+        res = super()._get_payment_method_information()
+        res['new_third_checks'] = {'mode': 'multi', 'domain': [('type', '=', 'cash')]}
+        res['in_third_checks'] = {'mode': 'multi', 'domain': [('type', '=', 'cash')]}
+        res['out_third_checks'] = {'mode': 'multi', 'domain': [('type', '=', 'cash')]}
+        return res
```

Comment:
Not sure why the difference, but the `check_printing` method has a domain on `bank` instead of `cash`

Reply:
third party checks are considered as cash. So using them in cash journals is more natural.
on the other hand, own checks are really another way of paying from bank.
I believe this is ok.

Thanks again for the deep review

Reply:
@william-andre indeed as Juan said, in this case, checks are treated as "cash" but that will be collected in the future. 
So this is ok
thanks!
cc' @jjscarafia @jco-odoo